README.md

```diff
@@ -130,32 +142,36 @@ A simplified manifest would look like this:
   "name": "My Company",
   "implementation": "My implementation",
   "issuers": [{
-    "id": "",
+    "id": "did:key:myIssuer1#keyFragment",
     "endpoint": "https://mycompany.example/credentials/issue",
     "method": "POST",
     "supportedEcdsaKeyTypes": ["P-256", "P-384"]
     "tags": ["ecdsa-rdfc-2019"]
   }, {
-    "id": "",
+    "id": "did:key:myIssuer#issuer2",
     "endpoint": "https://mycompany.example/credentials/issue",
     "method": "POST",
     "supportedEcdsaKeyTypes": ["P-256"]
     "tags": ["ecdsa-jcs-2019"]
   }, {
-    "id": "",
+    "id": "did:key:myIssuer3",
     "endpoint": "https://mycompany.example/credentials/issue",
     "method": "POST",
     "supportedEcdsaKeyTypes": ["P-256"]
     "tags": ["ecdsa-sd-2023"]
   }],
   "verifiers": [{
-    "id": "",
     "endpoint": "https://mycompany.example/credentials/verify",
     "method": "POST",
     "supportedEcdsaKeyTypes": ["P-256", "P-384"]
     "tags": [
       "ecdsa-rdfc-2019", "ecdsa-jcs-2019", "ecdsa-sd-2023"
     ]
+  }],
+  "vcHolders": [{
+    "id": "did:key:myIssuer1#keyFragment",
+    "endpoint": "https://mycompany.example/credentials/derive",
+    "tags": ["vcHolder"]
```

Comment:
Should this metadata be associated with only the `ecdsa-sd-2023` issuer rather than appear here on it's own?

Reply:
This design is not that best and I think this could be accomplished with a compound tag like `tags: ['ecdsa-sd-2023', 'derive']` and then just list it in the issuers section, but this PR is about documentation and also once the local issuer is in, we can possibly drop this tag entirely. So maybe an issue about vcHolder and the issues around it would be better? maybe all it needs is the `ecdsa-sd-2023` in addition to the `vcHolder` in the manifest?